spec/requests/core_induction_programme/core_induction_programme_spec.rb

```diff
@@ -4,7 +4,12 @@
 
 RSpec.describe "Core Induction Programme", type: :request do
   it "renders the core_induction_programme page" do
-    get "/core_induction_programme"
+    get "/core-induction-programme"
     expect(response).to render_template(:show)
   end
+
+  it "redirects to cip path when user is not admin" do
```

Comment:
Naturally, I'd expect a corollary spec for when a user is an admin

Reply:
So would I.

Turns out it's not trivial to test `send_file` method. Internet tells me to 'just stub it' in a controller test, but I didn't get it to work easily. Do you have any ideas on how we can handle it?

Reply:
Did you try: response.body.should eql data
response.header['Content-Type'].should eql 'image/png'

Reply:
I got it to work, the problem was that tests didn't want to generate the file in the place I wanted them to. After a long attempt at stubbing that file out I gave up and decided to just check it in.